tools/make-pkgs/make-pkgs.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+#VARDOC
+#FILE_DIR_OBK (PERSIST) : General path where the scripts will store config, git repos etc etc
+#DISTRO_COUNT (TEMP) : Counter to check if user tries to compile for multiple distros without --toolbox flag
+#config.bash : As toolbox containers donot inherit any environment variables, PERSISTENT variables are stored in here and loaded into other .sh files using source command
+#IM_*_OBK, TOOLBOX_ENABLE_OBK, <DISTRO>*_OBK (PERSIST) : self-explanatory
+#cleanup : cleans FILE_DIR_OBK and removes containers created by the script
+#help : shows helpinfo
+
+# Guide to add new distros
+# 1. add distro switch to the #ARGUMENT HANDLING section of make-pkgs.sh and declare variables appropriately
+# 2. add appropriate distro-specific commands to #TOOLBOX USAGE section
+# 3. add appropriate distro-specific commands to #TOOLBOX NO USAGE section
+# 4. add appropriate make-<DISTRO>.sh file (optional but recommended)
+# 5. call make-build.sh with appropriate arguments
+
+#Initial vars
+FILE_DIR_OBK="$HOME/.obk-build/"
+mkdir "$FILE_DIR_OBK" > /dev/null 2>&1
+rm -r "$FILE_DIR_OBK"config.bash > /dev/null 2>&1
+touch "$FILE_DIR_OBK"config.bash > /dev/null 2>&1
+echo "START_DIR_OBK=$(pwd)" >> "$FILE_DIR_OBK"config.bash
+#rm -r "$FILE_DIR_OBK"distro.list > /dev/null 2>&1
+#touch "$FILE_DIR_OBK"distro.list > /dev/null 2>&1
+
+#cleanup function
+cleanup () {
+  echo "Removing obk-toolbox containers..."
+  for tbc in $(toolbox list | sed -n 's/.*\(obk-toolbox[^\ ]*\).*/\1/p'); do
+    toolbox rm -f "$tbc"
+  done
+  echo "Cleaning build directory..."
+  rm -rf "$FILE_DIR_OBK" > /dev/null 2>&1
+}
+#help function
+help () {
+    echo "Usage: $0 [OPTIONS]"
+    echo "Options:"
+    echo "  --ibus          Compile IBus version"
+    echo "  --fcitx         Compile Fcitx version (Doesn't work for OBK2 and earlier)"
+    echo "  --develop       Compile from the develop branch"
+    echo "  --fedora        Compile for Fedora"
+    echo "  --debian        Compile for Debian"
+    echo "  --toolbox       Compile inside of a toolbox"
+    echo "  --clean         Clean \"$FILE_DIR_OBK\" directory and remove obk-toolbox-* containers"
+    echo "  --help          Display this help message"
+    echo " "
+    echo "  -> NOTE: User needs to specify the --toolbox flag to compile for multiple distros at the same time."
+    echo "           Otherwise, only one flag can be used at a time (e.g. --fedora and --debian can't be used together without --toolbox)"
+    echo "  -> NOTE: When compiling inside of a toolbox, an additional <version> can be specified for the distro flags."
+    echo "           (e.g. --fedora 37) This will generate packages for a specific version of a distro."
+    echo "Example: "
+    echo "  $0 --ibus --fcitx --develop --fedora 37 --debian --toolbox"
+    echo "  $0 --ibus --develop --debian"
```

Comment:
Instead of using multiple `echo`s, try using
```sh
cat <<EOF
...
EOF
```

Reply:
Unless absolutely mandatory, I would personally like to avoid the change, as using the echos make the script more readable and understandable compared to using cat <<EOF